loader/Dockerfile

```diff
@@ -21,5 +21,5 @@ ENV RELEASE="${RELEASE}"
 ENV API_URL=""
 ENV API_KEY=""
 
-# run our built server
+# Run the loader
```

Comment:
The previous comment here was never accurate, but I saw it when trying to make sure I’d removed all references to the server and figured I may as well fix it.